notes.md

```diff
@@ -49,9 +49,9 @@ Encryption information is stored in the `ContentEncodings Element` under the `Co
 
 # Image cropping
 
-Thanks to the PixelCropXXX elements, it's possible to crop the image before being resized. That means the image size follows this path:
+Thanks to the `PixelCrop Elements` (`PixelCropTop`, `PixelCropBottom`, `PixelCropRight` and `PixelCropLeft`), it's possible to crop the image before being resized. The image size follows this pattern, using `PixelCropTop` as an example:
 
-PixelXXX (size of the coded image) -> PixelCropXXX (size of the image to keep) -> DisplayXXX (resized cropped image)
+`PixelHeight` (size of the coded image) -> `PixelCropTop` (size of the image to keep) -> `DisplayHeight` (resized cropped image)
```

Comment:
We should be precise here and state that the value of `PixelCropTop` must be subtracted from `PixelHeight` in order to calculate the height of the cropped image. Or maybe make it even more explicit including all four values:

> `PixelWidth` (width of the coded image) → remove `PixelCropLeft` pixels from the left and `PixelCropRight` pixels from the right (size of the cropped image to keep) → resize to `DisplayWidth` (width of cropped & resized image)
> `PixelHeight` (size of the coded image) → remove `PixelCropTop` pixels from the top and `PixelCropBottom` pixels from the bottom (size of the cropped image to keep) → resize to `DisplayHeight` (height of cropped & resized image)

Doesn't sound that nice either, but I'd like to have state what a player must do a bit more explicitly than your change proposes.